docs/wiki/introduction/sql.md

```diff
@@ -499,6 +528,87 @@ There are also encoding functions available, to process query results.
     </p>
     </details>
 
+#### Collations
+
+- `version`:
+
+    <details>
+    <summary>Version collation example:</summary>
+    <p>
+
+      osquery> .mode line
+
+      osquery> select '1.0' = '1.0' collate version;
+      '1.0' = '1.0' collate version = 1
+
+      osquery> select '50.4.1b' < '50.4.1c' collate version;
+      '50.4.1b' < '50.4.1c' collate version = 1
+
+      osquery> select '20.10a' > '20.102' collate version;
+      '20.10a' > '20.102' collate version = 1
+    </p>
+    </details>
+
+- `version_arch`:
+
+    <details>
+    <summary>Version ARCH collation example:</summary>
+    <p>
+
+      osquery> .mode line
+
+      osquery> select '4:2' = '4:2-1' collate version_arch;
+      '4:2' = '4:2-1' collate version_arch = 1
+
+      osquery> select '2-2pre' < '2-2rc' collate version_arch;
+      '2-2pre' < '2-2rc' collate version_arch = 1
+
+      osquery> select '42.2-1' > '42.1-2' collate version_arch;
+      '42.2-1' > '42.1-2' collate version_arch = 1
+
+    </p>
+    </details>
+
+- `version_dpkg`:
+
+    <details>
+    <summary>Version DPKG collation example:</summary>
+    <p>
+
+      osquery> .mode line
+
+      osquery> select '1:2.0-10' = '1:2.0-10' collate version_dpkg;
+      '1:2.0-10' = '1:2.0-10' collate version_dpkg = 1
+
+      osquery> select '22.07.5-2ubuntu1.3' < '22.07.5-2ubuntu1.4' collate version_dpkg;
+      '22.07.5-2ubuntu1.3' < '22.07.5-2ubuntu1.4' collate version_dpkg = 1
+
+      osquery> select '2:8.2.3995-1ubuntu2.9' > '2:8.2.3995-1ubuntu2.3' collate version_dpkg;
+      '2:8.2.3995-1ubuntu2.9' > '2:8.2.3995-1ubuntu2.3' collate version_dpkg = 1
+
+    </p>
+    </details>
+
+- `version_rhel`:
```

Comment:
you think `rhel` not `rpm`? I guess maybe some of redhat is moving away from rpm

Reply:
That and it could be rpm, dnf, or yum.